<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>registerWidget</class>
    <widget class="QDialog" name="registerWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="maximumSize">
            <size>
                <width>800</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>用户注册</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>130</y>
                    <width>101</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>新用户注册</string>
            </property>
        </widget>
        <widget class="QWidget" name="formWidget" native="true">
            <property name="geometry">
                <rect>
                    <x>210</x>
                    <y>170</y>
                    <width>371</width>
                    <height>231</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                    <number>25</number>
                </property>
                <property name="verticalSpacing">
                    <number>25</number>
                </property>
                <property name="topMargin">
                    <number>11</number>
                </property>
                <item row="0" column="0">
                    <widget class="QLabel" name="label_2">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>请输入用户名*</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QLineEdit" name="inputUserName">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QLabel" name="label_3">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>请输入密码*</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="1">
                    <widget class="QLineEdit" name="inputUserPwd1">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
                <item row="2" column="0">
                    <widget class="QLabel" name="label_4">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>重复输入密码*</string>
                        </property>
                    </widget>
                </item>
                <item row="3" column="0">
                    <widget class="QLabel" name="label_5">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>邮箱*</string>
                        </property>
                    </widget>
                </item>
                <item row="3" column="1">
                    <widget class="QLineEdit" name="inputEmail">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Normal</enum>
                        </property>
                    </widget>
                </item>
                <item row="2" column="1">
                    <widget class="QLineEdit" name="inputUserPwd2">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QPushButton" name="submitUser">
            <property name="geometry">
                <rect>
                    <x>350</x>
                    <y>430</y>
                    <width>93</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>确定</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
